common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -77,7 +77,7 @@ public String toString() {
 
         if (logger.isDebugEnabled()) {
             if (DEFAULT_MAX_CAPACITY_PER_THREAD == 0) {
-                logger.debug("-Dio.netty.recycler.maxCapacityPerThread: disabled");
+                logger.debug("-Dsiren.io.netty.recycler.maxCapacityPerThread: disabled");
                 logger.debug("-Dio.netty.recycler.ratio: disabled");
                 logger.debug("-Dio.netty.recycler.chunkSize: disabled");
                 logger.debug("-Dio.netty.recycler.blocking: disabled");
```

Comment:
Should those also be prefixed with `siren.` ?

Reply:
No, we only prefix the `maxCapacityPerThread` with `siren.`